HtmlBuilderCore/Elements/HtmlTable.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HtmlBuilderCore.Elements
+{
+    public class HtmlTable : IHtmlElement
+    {
+
+        public int rows { get; set; }
+        public int columns { get; set; }
+        public string values { get; set; }
+        public List<object> list { get; set; }
+        int i, j;
```

Comment:
The `i` and `j` variables don't need to properties of the class. (right now, because you put them here, you've defined them as a private properties, eg you can do : `new HtmlTable().i`